plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php

```diff
@@ -79,7 +79,7 @@ public function get_title() {
 	public function get_content() {
 		return self::can_use_automated_taxes()
 			? __(
-				'Good news! WooCommerce Services and Jetpack can automate your sales tax calculations for you.',
+				'Good news! WooCommerce Tax can automate your sales tax calculations for you.',
```

Comment:
This isn't really visible anywhere aside from the API response. I figured let's change it just in case these become visible in the future.

![Markup on 2023-09-06 at 00:16:49](https://github.com/woocommerce/woocommerce/assets/1759681/f3df519f-2a90-48d2-9711-47e5ca68780f)